integrationtests/utils.go

```diff
@@ -27,7 +27,7 @@ import (
 
 // startServer starts the proxy's GRPC servers, and returns a function to shut them down when done with testing
 func startServer(t *testing.T, apiGroups ...srvtypes.APIGroup) func() {
```

Comment:
Would it be possible to add an integration test that makes a request to the new `metrics/` endpoint?